apiextensions/storageversion/migrator_test.go

```diff
@@ -88,6 +103,22 @@ func TestMigrate(t *testing.T) {
 	)
 }
 
+func TestMigrate_SingleStoredVersion(t *testing.T) {
+	// setup
+	dclient := dynamicFake.NewSimpleDynamicClient(runtime.NewScheme())
+	cclient := apixFake.NewSimpleClientset(singleVersionFakeCRD)
+	m := NewMigrator(dclient, cclient)
+
+	if err := m.Migrate(context.Background(), fakeGR); err != nil {
+		t.Fatal("Migrate() =", err)
+	}
+
+	assertPatches(t, cclient.Actions(),
+		// patch resource definition dropping non-storage version
+		crdStorageVersionPatch(singleVersionFakeCRD.Name, "v1"),
```

Comment:
assume if the storage version remains the same we don't need to patch the CRD status